assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -214,6 +214,47 @@ const AddressForm = ( {
 					);
 				}
 
+				const customValidationHandler = (
+					inputObject: HTMLInputElement
+				) => {
+					if (
+						! isPostcode( {
+							postcode: values.postcode,
```

Comment:
Are we using `values` state because `inputObject.value` only gives us postcode? 

I wonder if the rest of the address state is something we should pass to `customValidationHandler` to make this type of validation easier. cc @senadir ?

Reply:
I'm not really sure, I think it make sense to only `inputObject` for now, as long as we're not using the address is some comparison or event tracking then it's fine for now.

Reply:
> Are we using values state because inputObject.value only gives us postcode?

I was using `values` as the callback function for `customValidation` expects an `HTMLInputElement` as input. 

> I wonder if the rest of the address state is something we should pass to customValidationHandler to make this type of validation easier.

Within `customValidationHandler()`, `values` holds the following object:

```json
{
    "first_name": "",
    "last_name": "",
    "company": "",
    "address_1": "",
    "address_2": "",
    "city": "111",
    "state": "",
    "postcode": "",
    "country": "",
    "phone": ""
}
```

Is that what you meant?